docs/guides/using-the-network-shim.md

```diff
@@ -0,0 +1,193 @@
+# Using the Network Shim
+
+## About
+
+The Network Shim is a tool that provides a cypress plugin as well as a setup utilility for cypress' `setupFile`, which have to be used in conjunction. It uses the cypress `cy.intercept()` API to intercept network traffic during a test run. It has three different _modes_:
+
+-   `capture`: when operating in this mode, the Network Shim will intercept all outgoing requests to the specified hosts and allow each request to reach the destination server. Each response is stored as a fixture (JSON file).
+-   `stub`: in this mode the Network Shim also intercepts all outgoing requests to the specified hosts, but it will prevent the request from reaching the destination server and instead it will produce a response from a previously recorded fixture.
+-   `live`: the Network Shim does not intercept any requests when this mode is enabled.
+
+By first recording fixtures during a _capture run_ and then using these fixtures during a _stub run_ it effectively becomes possible to execute a full cypress test suite without a live backend. This is especially relevant for running a cypress test suite on CI.
```

Comment:
This relates to the intro text I suggested. Couple of tweaks:

> By recording fixtures during a _capture run_ and then using these fixtures during a _stub run_ you can execute a full cypress test suite without a live backend. This is especially useful when running cypress tests on CI. Since you no longer have to rely on a network connection and a backend, your cypress tests on CI can be more stable. One caveat is that your tests are slightly less realistic, since the network traffic is mocked, so you'll have to make sure that your recorded fixtures are kept up-to-date.